tools/ci_connector_ops/ci_connector_ops/pipelines/README.md

```diff
@@ -0,0 +1,81 @@
+# POC of CI connector pipelines in python
+
+This Python subpackage of `ci-connector-ops` gathers the POC code we're working on to:
+- Rewrite [airbyte-python.gradle](https://github.com/airbytehq/airbyte/blob/7d7e48b2a342a328fa74c6fd11a9268e1dcdcd64/buildSrc/src/main/groovy/airbyte-python.gradle) and [airbyte-connector-acceptance-test.gradle](https://github.com/airbytehq/airbyte/blob/master/buildSrc/src/main/groovy/airbyte-connector-acceptance-test.gradle) in Python.
+- Centralize the CI logic for connector testing 
+- Try out Dagger.io as a promising tool that can provide parallelism and caching out of the box for CI 
+
+## Install and use
+From `airbyte` repo root:
+
+### Install
+```bash
+cd tools/ci_connector_ops
+python -m venv .venv (please use Python 3.10)
+source .venv/bin/activate
+pip install -e .
+cd ../..
+```
+
+### Use
+
+### [Optional] Set your environment variables
+Optional: Set the DIFFED_BRANCH env variable. 
+This should be the branch name you want the internal git diff to run on to spot connector changes.
+```bash
+export DIFFED_BRANCH=master #This is already the default
+```
+
+Optional: Set your `GCP_GSM_CREDENTIALS` env variable. **If you don't set it you won't be able to run acceptance tests on connectors that uses secrets from GSM.** 
+More details [here](https://github.com/airbytehq/airbyte/blob/master/tools/ci_credentials/README.md#L20).
+
+```bash
+export GCP_GSM_CREDENTIALS=`cat <path to service account json file>`
+```
+
+
+### **Run the pipelines for a specific connectors**
+(source-pokeapi does not require GSM access)
+```bash
+connectors-ci test-connector source-pokeapi
+```
+
+### **Run the pipeline for multiple connectors**
+
+```bash
+connectors-ci test-connector source-pokeapi source-openweather
+```
```

Comment:
Nice!

Reply:
By downloading the connector catalog we can easily create other nice commands like:
```bash
connectors-ci test-all-connectors
connectors-ci test-all-ga-connectors
connectors-ci test-all-beta-connectors
connectors-ci test-all-alpha-connectors
```

The command structure is not definitive and can easily be revamped to our liking, `e.g`:
```
connectors-ci test-connectors --language python --language low-code --release-stage beta
connectors-ci test-connectors --name source-pokeapi --name source-openweather
```